packages/@react-spectrum/calendar/src/CalendarBase.tsx

```diff
@@ -106,6 +115,15 @@ export function CalendarBase<T extends CalendarState | RangeCalendarState>(props
       <div className={classNames(styles, 'spectrum-Calendar-months')}>
         {calendars}
       </div>
+      {state.validationState === 'invalid' &&
+        <HelpText
+          showErrorIcon
+          errorMessage={props.errorMessage || formatMessage('invalidSelection', {selectedCount: 'highlightedRange' in state ? 2 : 1})}
+          errorMessageProps={errorMessageProps}
+          validationState="invalid"
+          // Intentionally a global class name so it can be targeted in DatePicker CSS...
+          UNSAFE_className="spectrum-Calendar-helpText" />
```

Comment:
Anyone have a betted idea? 😬

Reply:
Technically it should exist in both modules as its own declaration.

You've mitigated part of the risk by having a combined selector that puts this global selector as a descendent of a modularized selector.

Another option is that you might be able to import the CSS from one into the other, that may allow the classname to be resolved to the same modularized value

Reply:
I don't have anything better right now